DashPay/Presentation/Home/DWDashPayReadyProtocol.h

```diff
@@ -19,11 +19,11 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern NSNotificationName const DWDashPayAvailabilityStatusUpdatedNotification;
+//extern NSNotificationName const DWDashPayAvailabilityStatusUpdatedNotification; TODO: check if needed
 
 @protocol DWDashPayReadyProtocol <NSObject>
 
-@property (readonly, nonatomic, assign) BOOL isDashPayReady;
+- (BOOL)shouldShowCreateUserNameButton;
```

Comment:
Given that `isDashPayReadyValue` method was renamed to `shouldShowCreateUserNameButton` before, it might make sense to expose it instead.